c2/cuda/groth16_cuda.cu

```diff
@@ -492,6 +574,7 @@ RustError::by_value generate_groth16_proofs_c(const Assignment<fr_t> provers[],
                     return;
 
                 // batch addition L - on GPU
+                if (!l_skip_batch_add)
                 execute_batch_addition<bucket_t>(gpu, circuit0, num_circuits,
```

Comment:
Missing indentation [here and below of course]...

Reply:
It would be arguably appropriate to cover the adjacent `if (caught_exception)` by the condition. To denote that we check for caught_exception after lengthy operations. Not that it actually matters for performance here, but rather for readability...